### ISSUE TITLE
Unwanted debug output during webpack build since v3

### ISSUE THREAD

**Issue by LeoniePhiline** [csstools/postcss-color-functional-notation#4]
Hi!

Since v3, I am getting unwanted debug output while dong a webpack build:

```
[ 'hsl', false ] Func {
  raws: { before: '', semicolon: false, after: '' },
  name: 'var',
  type: 'func',
  isColor: false,
  isVar: true,
  nodes:
   [ Word {
       raws: [Object],
       value: '--page-background-hsl',
       type: 'word',
       isColor: false,
       isHex: false,
       isUrl: false,
       isVariable: true,
       parent: [Circular],
       source: [Object] } ],
  parent:
   Func {
     raws: { before: ' ', semicolon: false, after: '' },
     name: 'hsl',
     type: 'func',
     isColor: true,
     isVar: false,
     nodes: [ [Circular], [Operator], [Numeric] ],
     parent:
      Func {
        raws: [Object],
        name: 'linear-gradient',
        type: 'func',
        isColor: false,
        isVar: false,
        nodes: [Array],
        parent: [Root],
        source: [Object],
        params:
         '(180deg, hsl(var(--page-background-hsl) / 20%) 1.14%, var(--page-background-color) 60%)',
        lastEach: 1,
        indexes: [Object] },
     source: { start: [Object], input: [Input], end: [Object] },
     params: '(var(--page-background-hsl) / 20%)' },
  source:
   { start: { line: 1, column: 1 },
     input:
      Input {
        css: 'var(--page-background-hsl) / 20%',
        hasBOM: false,
        id: '<input css 4>' },
     end: { line: 1, column: 4 } },
  params: '(--page-background-hsl)' }
[ 'hsl', false ] Func {
  raws: { before: '', semicolon: false, after: '' },
  name: 'var',
  type: 'func',
  isColor: false,
  isVar: true,
  nodes:
   [ Word {
       raws: [Object],
       value: '--page-background-hsl',
       type: 'word',
       isColor: false,
       isHex: false,
       isUrl: false,
       isVariable: true,
       parent: [Circular],
       source: [Object] } ],
  parent:
   Func {
     raws: { before: '', semicolon: false, after: '' },
     name: 'hsl',
     type: 'func',
     isColor: true,
     isVar: false,
     nodes: [ [Circular] ],
     parent:
      Root {
        raws: [Object],
        type: 'root',
        nodes: [Array],
        source: [Object],
        toString: [Function: bound toString],
        lastEach: 1,
        indexes: [Object] },
     source: { start: [Object], input: [Input], end: [Object] },
     params: '(var(--page-background-hsl))' },
  source:
   { start: { line: 1, column: 1 },
     input:
      Input {
        css: 'var(--page-background-hsl)',
        hasBOM: false,
        id: '<input css 269>' },
     end: { line: 1, column: 4 } },
  params: '(--page-background-hsl)' }
[ 'hsl', false ] Func {
  raws: { before: '', semicolon: false, after: '' },
  name: 'var',
  type: 'func',
  isColor: false,
  isVar: true,
  nodes:
   [ Word {
       raws: [Object],
       value: '--page-background-hsl',
       type: 'word',
       isColor: false,
       isHex: false,
       isUrl: false,
       isVariable: true,
       parent: [Circular],
       source: [Object] } ],
  parent:
   Func {
     raws: { before: '', semicolon: false, after: '' },
     name: 'hsl',
     type: 'func',
     isColor: true,
     isVar: false,
     nodes: [ [Circular] ],
     parent:
      Root {
        raws: [Object],
        type: 'root',
        nodes: [Array],
        source: [Object],
        toString: [Function: bound toString],
        lastEach: 1,
        indexes: [Object] },
     source: { start: [Object], input: [Input], end: [Object] },
     params: '(var(--page-background-hsl))' },
  source:
   { start: { line: 1, column: 1 },
     input:
      Input {
        css: 'var(--page-background-hsl)',
        hasBOM: false,
        id: '<input css 372>' },
     end: { line: 1, column: 4 } },
  params: '(--page-background-hsl)' }
```

I was wondering if this was left in there accidentally.

**Comment by Semigradsky** [comment]
https://github.com/csstools/postcss-color-functional-notation/blob/f2bcd1aa85d93d848105a4f6e42dc3fa8911153d/src/onCSSFunction.js#L14

@jonathantneal looks like it can be removed?

**Comment by romainmenke** [comment]
This was fixed in v4.0.0
